{{cookiecutter.repostory_name}}/app/src/{{cookiecutter.django_project_name}}/{{cookiecutter.django_default_app_name}}/email.py

```diff
@@ -26,24 +26,28 @@ def create_attachment(path: str, mime_type: MIMEType = MIMEImage) -> MIMEType:
 
 
 def send_mail(
-    template_name: str,
     subject: str,
-    body: str,
     to: list[str],
     from_: str = f'<{settings.DEFAULT_FROM_EMAIL}>',
+    cc: Optional[list[str]] = None,
+    template_name: Optional[str] = None,
+    body: Optional[str] = None,
     context: Optional[dict] = None,
     attachments: Optional[list[str]] = None,
 ):
```

Comment:
This is a breaking change if the consumer passes args by position. While one should review template changes when merging, it'd be good to avoid breaking changes if they're not truly necessary.
Though perhaps this is acceptable if we check which of our project use this utility and fix it immediately?

Reply:
I have no idea about other projects using this, and this is a relatively fresh change (1 week). @agoncharov-reef – as the original creator, maybe you know of a place that currently uses this code?

Reply:
Yeah, my bad - forgot to remove the `body`. I see you left it. Do we need it at all? I think it's a good thing to *force* people to use templates instead of putting email body right in python code.

Reply:
That's a fair point. I just wanted to be lazy in RAT, as there is no template and the whole HTML is rendered inside the code (and is as plain as it gets). But I can just remove the body, simplify the rest and roll with it. The only real thing I needed from it, was the `cc` parameter.

Reply:
I'll remove the body altogether then.

Reply:
Done.